add_model_instructions.md

```diff
@@ -125,3 +126,5 @@ This README provides instructions for deploying a trained model using the **`nob
 ## **Creating a Pull Request**
 
 After testing your model locally, you can create a pull request with your trained-models forked repository to merge your changes into the main **`trained-models`** repository.
+
+To do so, when creating a PR request, you will be prompted to fill in a template. Fill in the template with the required information, and remember that in 'Organization Name' you should put your model's parent folder since that is the organization name of your model (<org>/<model_name>/<version>).
```

Comment:
I reckon we should also ask for a version number.

Reply:
I agree so that we can find the new model files without crawling the folders looking for a Dockerfile.